recipes/hoggormplot/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "hoggormplot" %}
+{% set version = "0.13.2" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/hoggormplot-{{ version }}.tar.gz
+  sha256: cc34395e11a8b66b62c3eedd37dd9b2ce987f75b4e041122728ebb7d8a95bb51
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+    - sphinx
+    - hoggorm >=0.13.2
+    - matplotlib >=2.1.1
+  run:
+    - numpy >=1.9
+    - python >=3.6
+    - hoggorm >=0.13.2
+    - matplotlib >=2.1.1
```

Comment:
These should be `matplotlib-base`

Reply:
thank you, I updated the recipe

Reply:
I think the recipe could be merged? or is there anything else to do/improve?